styles.css

```diff
@@ -1,7 +1,73 @@
-h1 {
-  color: red;
+.flex-container {
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
 }
 
-p {
-  color: blue;
+nav {
+  height: 80px;
+  width: 100%;
+  font-size: 30px;
+}
+
+nav label {
+  float: right;
+  line-height: 80px;
+  position: relative;
+  top: 50px;
+  right: 70px;
+}
+
+.logo {
+  position: relative;
+  float: left;
+  top: 70px;
+  left: 20px;
+  font-weight: bold;
+}
+
+section h1 {
```

Comment:
- [ ] Please consider adding the `background-image` to your `section` tag. You will notice that in the Figma design you chose, there are some cool graphics or images 🎨 that you have not included in your design while they play an important role in making your portfolio looking awesome and attractive 😃.
Figma design
![image](https://user-images.githubusercontent.com/90222110/176926406-edfdfc9b-62f0-4e96-89cd-b41bcb5fae68.png)
Your design
![image](https://user-images.githubusercontent.com/90222110/176926616-8f4f306a-7121-43a9-a517-331288f11c27.png)

Reply:
I have tried to add graphics like Figma template, unfortunately, I failed to add one-by-one vectors to section tag.
please suggest to me any tutorial article on how to add vector.